### ISSUE TITLE
Error when building Wasm on OSX

### ISSUE THREAD

**Issue by johba37** [TrueBitFoundation/ocaml-offchain#96]
i went through similar instructions on High Sierra:
```
brew install opam ocaml
opam init -y
eval $(opam config env)
opam install cryptokit yojson
```

Then encountered this error when executing `make` to build wasm:

```
# Error: This expression has type bytes but an expression was expected of type
#          string
```

**Comment by johba37** [comment]
looked up the error here: https://github.com/facebook/flow/issues/5270

```
export OCAMLPARAM="safe-string=0,_"
```

solved the issue, all tests passing.

**Comment by mrsmkl** [comment]
Hopefully this is fixed in #97

**Comment by mrsmkl** [comment]
I guess this has now been fixed, we use a later version of OCaml now
